modules/lofreq/1.0/src/lofreq_filter.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
What's the rationale for this change? I always use `#!/bin/bash` in my scripts, curious what the difference is.